src/routes/Router/Router.tsx

```diff
@@ -9,8 +9,11 @@ import { ChooseOrgPage } from '@/features/apiDelegation/offered/ChooseOrgPage';
 import { ReceiptPage } from '@/features/apiDelegation/offered/ReceiptPage';
 import { ConfirmationPage } from '@/features/apiDelegation/offered/ConfirmationPage';
 import { NotFoundSite } from '@/sites/NotFoundSite';
+import { OverviewPage } from '@/features/singleRight/offered/OverviewPage/OverviewPage';
```

Comment:
```suggestion
import { OverviewPage as SingleRightsOveview } from '@/features/singleRight/offered/OverviewPage/OverviewPage';
```
That being said.. Are we even gonna have an overview page on single rights? 🤔

Reply:
Don't know what to call it yet, I thought the first page could be called overview. Maybe I should change it to StartPage? Open for suggestions!

Reply:
We start right on the delegation, don't we?
Maybe chooseService? And then the next one we can call specifyRights?